src/Symfony/Component/Finder/Iterator/RecursiveDirectoryIterator.php

```diff
@@ -31,6 +31,10 @@ class RecursiveDirectoryIterator extends \RecursiveDirectoryIterator
      */
     private $rewindable;
 
+    private $rootPath;
+    private $subPath;
+    private $directorySeparator = '/';
```

Comment:
please add some documentation on them to make the code understandable by someone reading the code later